docs/basics.txt

```diff
@@ -1,25 +1,21 @@
 [[basics]]
-JanusGraph Basics
-=================
+= JanusGraph Basics
 
 [[configuration]]
-Configuration
--------------
+== Configuration
 
-A JanusGraph graph database cluster consists of one or multiple JanusGraph instances. To open a JanusGraph instance a configuration has to be provided which specifies how JanusGraph should be set up.
+A JanusGraph server cluster consists of one or multiple JanusGraph server instances. To open a JanusGraph instance, a configuration has to be provided which specifies how JanusGraph should be set up.
```

Comment:
Reverting this change as this will need further discussion; added an entry to the deferred docs update issue.